src/gevent/monkey.py

```diff
@@ -357,11 +367,13 @@ def _patch_module(name,
                   _warnings=None,
                   _notify_will_subscribers=True,
                   _notify_did_subscribers=True,
-                  _call_hooks=True):
+                  _call_hooks=True,
+                  _patch_module=False,
+                  _package_prefix='gevent.'):
 
-    gevent_module = getattr(__import__('gevent.' + name), name)
+    gevent_module = import_module(_package_prefix + name)
```

Comment:
Probably better named `source_module` at this point.

Reply:
I have no problem with any aesthetic API changes (argument requirement, order, naming, refactoring).